deployment/kubernetes/agents/base/agent.yaml

```diff
@@ -16,7 +16,7 @@ spec:
     spec:
       serviceAccountName: stackstate-agent
       containers:
-      - image: docker.io/stackstate/stackstate-agent-2-test:master
+      - image: docker.io/stackstate/stackstate-agent-test:master
```

Comment:
if i understand correctly we consolidated the test repos for both py2 py3

Reply:
docker test repo, yes
for the other distributions (deb, rpm, msi) we keep separate buckets